### ISSUE TITLE
[Bug] v.to.db check if column exists broken

### ISSUE THREAD

**Issue by jradinger** [OSGeo/grass#384]
When running `v.to.db` in the GRASS 7.9dev causes an error related to the creating of new columns. More specifically, I've predefined columns in an attribute table (linked to layer 2) which I want to update with the start and end coordinates of the lines. I am aware that there is a new feature related to v.to.db that checks if a column already exists and creates the columns if the do not exists.

**Reproducible example**
Steps to reproduce the behavior based on the NC dataset and based on python3 commands:
```
grass.run_command("v.category",
	overwrite=True,
	input="busroute_a@PERMANENT",
	option="add",
	layer=2,
	output="busroute_a2")
grass.run_command("v.db.addtable",
	map="busroute_a2", layer=2,
	columns="start_x DOUBLE, start_y DOUBLE, end_x DOUBLE, end_y DOUBLE")
grass.run_command("v.to.db",
	map="busroute_a2", layer=2,
	option="start",
	columns="start_x, start_y")

```

**System description:**
GRASS version: 7.9.dev                                                          
Code revision: 8eba412c8                                                        
Build date: 2020-02-27                                                          
Build platform: x86_64-pc-linux-gnu                                             
GDAL: 3.1.0dev-c74609560e                                                       
PROJ: 6.2.1                                                                     
GEOS: 3.8.0                                                                     
SQLite: 3.29.0                                                                  
Python: 3.7.5                                                                   
wxPython: 4.0.6                                                                 
Platform: Linux-5.3.0-40-generic-x86_64-with-Ubuntu-19.10-eoan

**Comment by metzm** [comment]
The bug is in the call of v.to.db: `columns="start_x, start_y"`. The second column's name is " start_y", not "start_y". Removing the leading whitespace from " start_y" works.

**Comment by jradinger** [comment]
Correct, there is an issue with white space. I am wondering if it would make sense to remove leading white spaces automatically (or raise an error message that the column name is not allowed). It seems that leading white space in not a problem in `v.db.addtable` as shown in the example above (`columns="start_x DOUBLE, start_y DOUBLE, end_x DOUBLE, end_y DOUBLE"`). Maybe this should also be adopted for consistency also in `v.to.db`.

**Comment by metzm** [comment]
Please test feecd4b which removes leading and trailing white spaces from column names.

**Comment by HuidaeCho** [comment]
Please check my comment at https://github.com/OSGeo/grass/commit/feecd4b7d47ba9481458656f84387903205a2733#commitcomment-37972472. This fix causes inconsistency.

**Comment by HuidaeCho** [comment]
@metzm Reiterating my concern about this "special" treatment of white spaces in just one module. Not all modules strip out leading/trailing white spaces. It should be either fixed globally or should be left as *designed* (i.e., a user error).

This commit causes inconsistency in handling white spaces in other modules. For example, `g.remove type=vector name="vector1, vector2"` (with a space) fails with
```
WARNING: Illegal filename < vector2>. Character < > not allowed.
ERROR: Illegal filenames not allowed in the name or ignore option.
```
Then, would it be a good idea to always remove white spaces from `lib/gis/parser.c` to make this effect global and consistent? I don't think so because some modules may need trailing white spaces (e.g., column values). I believe that this specific issue should be treated as a user error without code changes.
